.github/workflows/unit-test-on-pull-request.yml

```diff
@@ -77,13 +77,13 @@ jobs:
   check-binary-blobs:
     name: Check for differences in the eBPF and Rust binary blobs
     runs-on: ubuntu-24.04
-    container: otel/opentelemetry-ebpf-profiler-dev:latest
+    container: otel/opentelemetry-ebpf-profiler-dev:latest@sha256:688e4651a9a482700836ff59ea00270b2c237bbcb2cf6b6c9daffcf29fa0a0cb
```

Comment:
We probably shouldn't pin this?

Reply:
While it's considered best practice to pin dependencies in CI to their hash, I agree that the benefit here might be limited. It adds one additional step for us, if we update the image.
On the other hand, anyone with the correct permissions is able to overwrite this image atm and as we just use `latest` instead of a version, this might will help us identify such scenarios more easily.